systemvm/debian/opt/cloud/bin/cs/CsProcess.py

```diff
@@ -40,9 +40,9 @@ def kill_all(self):
 
     def find_pid(self):
         self.pid = []
+        items = len(self.search)
```

Comment:
maybe we should think about renaming search; it actually means to be the process/command name.